lib/tpm2_policy.c

```diff
@@ -392,6 +392,89 @@ bool tpm2_policy_build_policycommandcode(ESYS_CONTEXT *ectx,
     return true;
 }
 
+bool tpm2_policy_build_policylocality(ESYS_CONTEXT *ectx,
+    tpm2_session *session, uint8_t locality) {
```

Comment:
use the TPM defined type of `TPMA_LOCALITY`

Reply:
seems similar problems exists in https://github.com/tpm2-software/tpm2-tools/blob/2a5a90c34913ee7ae53f4f6987f08a978a2eb9be/lib/tpm2_policy.c#L382